From 4c327d4d5f3dd08bce7a704e2e4e45e2a838dc93 Mon Sep 17 00:00:00 2001
From: Pavel Valena <pvalena@redhat.com>
Date: Mon, 17 Feb 2025 02:50:26 +0100
Subject: [PATCH] fix(35network-manager): remove duplicate installkernel
 function

Issue introduced in commit: 0a264651d148b543c0c5d6b0a07909cdcb1abfba

Resolves: RHEL-64754

rhel-only
---
 modules.d/35network-manager/module-setup.sh | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/modules.d/35network-manager/module-setup.sh b/modules.d/35network-manager/module-setup.sh
index 4fd3d051..80ca21bd 100755
--- a/modules.d/35network-manager/module-setup.sh
+++ b/modules.d/35network-manager/module-setup.sh
@@ -14,11 +14,6 @@ depends() {
     return 0
 }
 
-# called by dracut
-installkernel() {
-    return 0
-}
-
 # called by dracut
 installkernel() {
     instmods nf_tables nfnetlink nft_fwd_netdev

